internal/gps/pkgtree/pkgtree.go

```diff
@@ -140,11 +142,19 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 			}
 		} else {
 			switch err.(type) {
-			case gscan.ErrorList, *gscan.Error, *build.NoGoError:
+			case *build.NoGoError:
+				return nil
+			case scanner.ErrorList:
+				el := err.(scanner.ErrorList)
+				var els []string
+
+				for _, e := range el {
+					els = append(els, fmt.Sprintf("%v", e))
+				}
 				// This happens if we encounter malformed or nonexistent Go
 				// source code
 				ptree.Packages[ip] = PackageOrErr{
-					Err: err,
+					Err: errors.Wrapf(err, "\n\t%v", strings.Join(els, "\n\t")),
```

Comment:
i think this is a change in a good direction, but it seems like we could use a sort of "header" row before the individual rows